node/resolvers/Mutations/Organizations.ts

```diff
@@ -337,12 +337,12 @@ const Organizations = {
       await masterdata.updatePartialDocument({
         dataEntity: ORGANIZATION_DATA_ENTITY,
         fields: {
-          name,
-          ...((tradeName || tradeName === '') && { tradeName }),
           collections,
+          ...((tradeName || tradeName === '') && { tradeName }),
+          customFields,
+          name,
```

Comment:
Are these order changes here required? How do they affect the solution?

Reply:
Ahhhh, I see, you're sorting alphabetically, right? I like it, it's just been so long since I've seen someone do this. I'm all for it! 👏  We can later look into a lint rule to enforce this, if the rest of the team also likes it 😬 .